test/parallel/test-http2-buffersize.js

```diff
@@ -16,21 +16,20 @@ const Countdown = require('../common/countdown');
   const server = createServer();
 
   let client;
-  const countdown = new Countdown(SOCKETS, () => {
-    client.close();
-    server.close();
-  });
 
-  // Other `bufferSize` tests for net module and tls module
-  // don't assert `bufferSize` of server-side sockets.
-  server.on('stream', mustCall((stream) => {
+  const getStream = async () => {
+    const [ stream ] = await once(server, 'stream');
     stream.on('data', mustCall());
     stream.on('end', mustCall());
+    stream.on('close', mustCall());
+    return once(stream, 'close');
+  };
 
-    stream.on('close', mustCall(() => {
-      countdown.dec();
-    }));
-  }, SOCKETS));
+  const promises = [...new Array(SOCKETS)].map(getStream);
```

Comment:
nit.. while in here it would be worthwhile to rename the constants to follow the idiomatic style.. s/SOCKETS/kSockets

Reply:
Sure, done!